src/clarity-angular/layout/tabs/chocolate/active-oompa-loompa.ts

```diff
@@ -12,9 +12,17 @@ import {TabsWillyWonka} from "./tabs-willy-wonka";
 
 @Directive({selector: "[clrTabLink], clr-tab-content"})
 export class ActiveOompaLoompa extends OompaLoompa {
-    constructor(cdr: ChangeDetectorRef, willyWonka: TabsWillyWonka, @Inject(IF_ACTIVE_ID) private id: number,
-                private ifActive: IfActiveService) {
+    private ifActive: IfActiveService;
+    private id: number;
+
+    constructor(cdr: ChangeDetectorRef, @Optional() willyWonka: TabsWillyWonka, @Inject(IF_ACTIVE_ID) id: number,
+                ifActive: IfActiveService) {
+        if (!willyWonka) {
+            throw new Error("clr-tab-content should only be used inside of a clr-tabs");
```

Comment:
`"clrTabLink and clr-tab-content should only be used inside of a clr-tabs"`

Reply:
Missed that. Thanks! Will get that fixed.